synapse/storage/schema/main/delta/67/01event_columns.sql

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2022 The Matrix.org Foundation C.I.C
```

Comment:
This file is in `67/` — should it be in `68/`?

Reply:
bah, when I first wrote this, 67 was the latest version :)